### ISSUE TITLE
Request for Anaconda Integration

### ISSUE THREAD

**Issue by phumke** [apache/airflow#1113]
AirFlow looks like it would be an amazing addition to my teams workflow, but most of my work is behind a firewall (and red tape) which makes installing dependencies a pain.  I'd love to see airflow (and it's dependencies) be included within [conda](https://www.continuum.io).

Thanks!

**Comment by patrick-bark** [comment]
+1 that would be really cool

**Comment by mistercrunch** [comment]
Can anyone request or does it have to be the maintainer of the project?

**Comment by patrick-bark** [comment]
> The GitHub repository at https://github.com/conda/conda-recipes is always happy to accept new recipes for programs not yet there.

**Comment by phumke** [comment]
Sweet!  Nice find.  I had meant to look into this last weekend, but I'll take an action to take a stab at building the necessary recipes in the next few days.

**Comment by patrick-bark** [comment]
Awesome man, report back on this whenever you can

**Comment by phumke** [comment]
Ok, I built the recipes for airflow and the dependencies that I couldn't find in an existing conda channel:

https://github.com/phumke/conda-recipes

And I uploaded the packages to conda:

https://anaconda.org/phumke

I haven't had a chance to test all of the packages out yet, so let me know if you run into any issues.  I know that gunicorn and setproctitle are fixing their windows support, so the win version of those probably don't work yet.

Also, there is a decent sized dependency tree for airflow that you'll have to download.  I have most of those listed in a text file, so if you want that list let me know and I can send that over.
